code/_onclick/observer.dm

```diff
@@ -14,7 +14,7 @@
 
 	// Otherwise jump
 	else if(A.loc)
-		loc = get_turf(A)
+		Move(get_turf(A.loc))

```

Comment:
uh no they don't they override a bunch of shit and only call crossed on step triggers